package.json

```diff
@@ -16,7 +16,8 @@
     "strdup": "0.0.0",
     "Constellation/console-colors.c": "1.0.1",
     "littlstar/asprintf.c": "0.0.3",
-    "logger": "0.0.0"
+    "logger": "0.0.0",
+    "kgabis/parson": "master"
```

Comment:
Sure! I apologize for not getting back to this PR, life got busy after the
holiday break. I'll make some time this week to polish it up.

On Mon, Jan 11, 2016 at 9:33 AM, Stephen Mathieson <notifications@github.com
> wrote:

> In package.json
> <https://github.com/clibs/clib/pull/124#discussion_r49350564>:
>
> > @@ -16,7 +16,8 @@
> >      "strdup": "0.0.0",
> >      "Constellation/console-colors.c": "1.0.1",
> >      "littlstar/asprintf.c": "0.0.3",
> > -    "logger": "0.0.0"
> > +    "logger": "0.0.0",
> > +    "kgabis/parson": "master"
>
> @darthtrevino <https://github.com/darthtrevino> i've forked the original
> parson repo and reverted the \/ nonsense. can you update to use
> clibs/parson@1.0.0 <https://github.com/clibs/parson/tree/1.0.0>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clibs/clib/pull/124/files#r49350564>.
>